cgmanifests/generated/cgmanifest.json

```diff
@@ -407,6 +397,77 @@
         },
         "comments": "cutlass"
       }
-    }
+    },
+    {
+      "component": {
+        "type": "git",
+        "git": {
+          "commitHash": "6f7ffeb18f99796233b958aaaf14ec7bd4fb64b2",
+          "repositoryUrl": "https://github.com/microsoft/vcpkg.git"
+        },
+        "comments": "vcpkg"
+      }
+    },
+    {
+      "component": {
+        "type": "git",
+        "git": {
+          "commitHash": "19cc035b6c6f2283573d29c7ea7f7d675cf750ce",
+          "repositoryUrl": "https://github.com/openssl/openssl.git"
+        },
+        "comments": "openssl"
+      }
+    },
+    {
+      "component": {
+        "type": "git",
+        "git": {
+          "commitHash": "f54b0e47a08782a6131cc3d60f94d038fa6e0a51",
+          "repositoryUrl": "https://github.com/Tencent/rapidjson.git"
+        },
+        "comments": "rapidjson"
+      }
+    },
+    {
+      "component": {
+        "type": "git",
+        "git": {
+          "commitHash": "da041154c6bac1a4aa98254a7d6819059e8ac0b0",
+          "repositoryUrl": "https://github.com/boostorg/boost.git"
+        },
+        "comments": "boost"
+      }
+    },
+    {
+      "component": {
+        "type": "git",
+        "git": {
+          "commitHash": "9a6546658657dbeb23245117b57f4e6cf6cdc3e6",
+          "repositoryUrl": "https://github.com/libb64/libb64.git"
+        },
+        "comments": "b64"
+      }
+    },
+    {
+      "component": {
+        "type": "git",
+        "git": {
+          "commitHash": "75a84807a019bf4961faf713df9d748f0fc83b47",
+          "repositoryUrl": "https://github.com/triton-inference-server/server.git"
+        },
+        "comments": "triton"
```

Comment:
do we also need to mention the other repos triton client depends on?

Reply:
Right - may be we should also add the client here.

Reply:
Client added.